### PR TITLE
[INLONG-822][Doc] Add the quick start guide for collecting file and writing to ClickHouse

### LINKED ISSUE #822: [Doc][Translate] Add the quick start guide for collecting file and writing to ClickHouse
### Description

There is a [Chinese document](https://mp.weixin.qq.com/s/7JvG32NJga-J1zYXiu-AJw). Contributors who want to work on this PR could refer to it.

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Amilelne**
Add the quick start guide for collecting file and writing to ClickHouse

fix #822

### CODE REVIEW

File: docs/quick_start/clickHouse_file_example.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: Collect File and Writing to ClickHouse Example
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_file_example.md:2]: File to ClickHouse

File: docs/quick_start/clickHouse_file_example.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: File to ClickHouse
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_file_example.md:2]: File -> Pulsar -> ClickHouse Example

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+title: 创建文件到 ClickHouse 数据流示例
+sidebar_position: 2
+---
+
+在下面的内容中，我们将通过一个完整的示例介绍如何使用 Apache InLong 创建文件到 ClickHouse 数据流。
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md:6]: 文件到 ClickHouse
->
`File -> Pulsar -> ClickHouse`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## 环境部署
+### 安装 InLong
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md:10]: Remove this empty line.

File: docs/quick_start/clickHouse_file_example.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
+---
+title: File to ClickHouse 
+sidebar_position: 2
+---
+
+Here we use an example to introduce how to collect file and write to clickHouse.
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_file_example.md:6]: collect file and write to clickHouse.
->
create `File -> Pulsar -> ClickHouse` data ingestion.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+![clickhouse](img/clickhouse/pulsar_table.png)
+
+## 常见问题
+### Docker Compose 启动出现“specify container image platform”
+API 不兼容导致，可以删除 docker-compose.yml 中的 platform: "linux/x86_64"，执行。
+
+### 数据流配置错误
+一般是 MQ 或者 Flink 集群配置错误导致，可以在页面查看错误信息，或者进入 Manager 容器查看详细日志。
+
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md:101]: Too many empty lines at the end.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_file_example.md:103]: Here lack the Q & A?

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
+---
+title: File -> Pulsar -> ClickHouse 示例
+sidebar_position: 2
+---
+
+在下面的内容中，我们将通过一个完整的示例介绍如何使用 Apache InLong 创建File -> Pulsar -> ClickHouse数据流。
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md:6]: 创建File -> Pulsar -> ClickHouse数据流
->
创建 File -> Pulsar -> ClickHouse 数据流

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+![Create Cluster label](img/clickhouse/create_cluster_label.png)
+
+**注：default_cluster 是各个组件默认上报集群标签，请勿使用其它名称**
+
+### 注册 Pulsar 集群
+页面点击 [集群管理]->[集群管理]->[新建集群]，注册 Pulsar 集群：
+![](img/create_pulsar_cluster.png)
+
+:::note
+集群标签选择刚创建的 `default_cluster`，docker 部署的 Pulsar 集群：
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md:37]: docker 部署的 Pulsar 集群
->
配置 Docker 部署的 Pulsar 集群：

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+:::note
+注：如果需要使用kafka，需要修改 docker-compose.yml 中的 MQ_TYPE。
+:::
+
+### 新建数据流
+点击【下一步】→【新建数据流】，配置 ID 及测试源字段信息：
+![Create datastream](img/clickhouse/pulsar_stream.png)
+
+### 为数据流添加文件数据源
+点击【新建数据源】，选择file类型，配置数据源信息，包括名称、选择采集 Agent 集群及 IP 、采集文件路径等：
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md:60]: 选择file类型
->
选择 File 类型

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+![Create datastream](img/clickhouse/pulsar_source.png)
+
+### 新建数据目标
+点击【新建数据目标】，选择ClickHouse类型，配置包括名称、库表名称、选择已创建的节点及 Schema 映射信息等，然后点击【提交审批】：
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md:64]: 选择ClickHouse类型
->
选择 ClickHouse 类型

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+页面点击 【数据节点】→【新建】，新增 ClickHouse 数据节点：
+![Create kafka cluster](img/clickhouse/datanode.png)
+
+## 任务创建
+### 新建数据流组
+页面点击【数据接入】→【新建接入】，只用填写 ID 接选择 Pulsar 即可：
+![Create datastream](img/clickhouse/pulsar_ingestion.png)
+
+:::note
+注：如果需要使用kafka，需要修改 docker-compose.yml 中的 MQ_TYPE。
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md:52]: 如果需要使用kafka
->
如果需要使用 Kafka

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
+---
+title: File -> Pulsar -> ClickHouse 示例
+sidebar_position: 2
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/file_pulsar_clickhouse_example.md:3]:
```suggestion
sidebar_position: 4
```

File: docs/quick_start/file_pulsar_clickhouse_example.md [hunk 14]
```diff
@@ -0,0 +1,101 @@
+---
+title: File -> Pulsar -> ClickHouse Example 
+sidebar_position: 2
```
Comment (dockerzhang) [inline docs/quick_start/file_pulsar_clickhouse_example.md:3]:
```suggestion
sidebar_position: 4
```

**Review by dockerzhang — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**